libnetwork/sandbox.go

```diff
@@ -98,6 +103,7 @@ type containerConfig struct {
 	useDefaultSandBox bool
 	useExternalKey    bool
 	exposedPorts      []types.TransportPort
+	sysctls           map[string]string
```

Comment:
My thinking was that the decision belongs to libnetwork, so it needs all the info - and, having a view of sysctl settings might be generally useful.

But, if I can move the logic back to between container create/start, this will go-away anyway.